api/capi/meson.build

```diff
@@ -27,13 +30,10 @@ capi_devel_main = []
 capi_devel_main += join_paths(meson.current_source_dir(), 'include', 'nnstreamer.h')
 capi_devel_main += join_paths(meson.current_source_dir(), 'include', 'nnstreamer-single.h')
 
-inc = include_directories('include')
-nninc = include_directories('../../gst')
-
 tizen_deps = []
 
 if (get_option('enable-tizen'))
-  message('CAPI is in Tizen mode')
+  message('capi is in tizen mode')
```

Comment:
Does it mean that message do not handle a capital? Why do we have to use lower-case?

Reply:
This is my mistake, I will revert and clean these changes.